fluster/decoders/gstreamer.py

```diff
@@ -111,13 +111,10 @@ def gen_pipeline(
             self.cmd, input_filepath, self.decoder_bin, self.caps, self.sink, output
         )
 
-    def parse_videocodectestsink_md5sum(self, data: List[str], verbose: bool) -> str:
+    def parse_videocodectestsink_md5sum(self, data: List[str]) -> str:
```

Comment:
What do you think about these changes (L117-L137)?

    def parse_videocodectestsink_md5sum(self, data: List[str]) -> str:
        """Parse the MD5 sum out of commandline output produced when using
        videocodectestsink."""
        # Define the pattern to search for in each line
        pattern = "conformance/checksum, checksum-type=(string)MD5, checksum=(string)"
        # Iterate over each line in the data
        for line in data:
            # Partition the line into three parts: before the pattern, the pattern, and after the pattern
            before, match, after = line.partition(pattern)
            # If the pattern is found (match is not empty)
            if match:
                # Partition the remaining part to find the checksum up to the first ';'
                checksum, _, _ = after.partition(";")
                # If a valid checksum is found
                if checksum:
                    return checksum

        raise Exception("No MD5 found in the program trace.")

Reply:
I'm going to change in another PR

Reply:
Thanks, and I agree that those changes are probably better to do in a separate PR, my changes where just bare minimum to move the verbose print to the utils function, where I think it belongs.